product-service/serverless.yml

```diff
@@ -45,8 +47,13 @@ provider:
 #                - "/*"
 
 # you can define service wide environment variables here
-#  environment:
-#    variable1: value1
+  environment:
+    PG_HOST: task4-db.cqa5v273sikm.eu-west-1.rds.amazonaws.com
+    PG_PORT: 5432
+    PG_DB: task4
+    PG_USER: postgres
+    PG_PWD: 123
+
```

Comment:
please do not store secrets in config file in repo - use .env
![Screenshot from 2022-07-31 12-49-23](https://user-images.githubusercontent.com/380645/182020675-d04154fa-1d6a-4d3f-8b4c-c79cfab9d529.png)

Reply:
meanwhile you've got max score 9 from my side :)